modelerfour/test/scenarios/body-datetime-rfc1123/flattened.yaml

```diff
@@ -81,14 +85,21 @@ schemas: !<!Schemas>
         name: date-time
         description: MISSING·SCHEMA-DESCRIPTION-DATETIME
     protocol: !<!Protocols> {}
-  numbers:
   - *ref_0
-  strings:
+  numbers:
   - *ref_1
+  strings:
+  - !<!StringSchema> &ref_3
```

Comment:
I see a lot of new `StringSchemas` being added, is this because other schemas that should have been of another type were being referenced elsewhere as `StringSchema` and now they're being promoted to the `schemas` list because the former schema is now a different schema (seems like `DateTimeSchema` in this case)?

Reply:
The delta new string schemas are me adding/using a common string type for a plain string. 

most of the rest is doc changes, and we're now using a real schema where we using string before,